test/e2e/ps_test.go

```diff
@@ -405,6 +405,19 @@ var _ = Describe("Podman ps", func() {
 		Expect(actual).ToNot(ContainSubstring("NAMES"))
 	})
 
+	It("podman ps filter by container command", func() {
+		session := podmanTest.Podman([]string{"run", "-d", "--name", "test1", ALPINE, "top"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(ExitCleanly())
+
+		result := podmanTest.Podman([]string{"ps", "-a", "--filter", "command=top"})
+		result.WaitWithDefaultTimeout()
+		Expect(result).Should(ExitCleanly())
+
+		output := result.OutputToStringArray()
+		Expect(output).To(HaveLen(2))
```

Comment:
Please check that the container ID of the first container is in the output

Reply:
Sure!